homeassistant/components/ista_ecotrend/quality_scale.yaml

```diff
@@ -0,0 +1,78 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: The integration registers no actions.
+  appropriate-polling: done
+  brands: done
+  common-modules:
+    status: done
+    comment: Has coordinator for centralized data retrieval. No need for base entity as there is only one platform.
```

Comment:
In the coordinator, a total of 3 executor jobs can be started for a single update. Switching from executor to the event loop is expensive, so it would be better to group them in one executor job